agri-store-react/src/Components/Context/ContextProductListingPage.js

```diff
@@ -28,7 +78,19 @@ function ProductListingPage({ children }) {
   return (
     <div>
       <ProductListingPagecontext.Provider
-        value={{ productdata, setProductData, isLoading }}
+        value={{
+          productdata,
+          setProductData,
+          isLoading,
+          addToWishlist,
+          addToCart,
+          cart,
+          setCart,
+          wish,
+          setWish,
+          updateQty,
+          deleteQty,
```

Comment:
Wishlist and cart should have separate context for better code logic division

Reply:
Yes this is bucket list after finishing the project will separate the logic for each page with respect to the context and reducer